config/charts/inferencepool/templates/epp-servicemonitor.yaml

```diff
@@ -0,0 +1,54 @@
+{{- if .Values.inferenceExtension.monitoring.servicemonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ include "gateway-api-inference-extension.name" . }}-monitor
```

Comment:
same comment, add namespace (I assume this is namespace scoped?)

Reply:
added